config/config_test.go

```diff
@@ -18,7 +18,7 @@ func TestParseTerragruntConfigDynamoLockMinimalConfig(t *testing.T) {
 	t.Parallel()
 
 	config :=
-`
+		`
```

Comment:
Agreed; I tried putting the opening backtick on the same line as the assignment operator.